packages/contracts/deploy/40_finalize-managing-dao/10_install-multisig-on-managing-dao.ts

```diff
@@ -0,0 +1,137 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
```

Comment:
This step seems to happen in the middle of the whole deployment

Why don't we install multisig (manual `grant()`) when the management DAO is created at the very beginning?

Otherwise, we will be using the PSP but we won't be able to test the DAO before rolling out the complete deployment